boards/nxp/rd_rw612_bga/Kconfig.defconfig

```diff
@@ -37,4 +37,7 @@ config NET_L2_ETHERNET
 
 endif # DT_HAS_NXP_ENET_MAC_ENABLED && NETWORKING
 
+config ZTEST_NO_YIELD
+	default y if (ZTEST && PM)
```

Comment:
Should it be set for all platforms?

Reply:
But for others it's at least set at soc level?

Reply:
@kartben, good point, let me put another PR to modify it. Thanks for checking